### PR TITLE
bench binary & trusted miners config & update interopnet

### LINKED ISSUE #375: cmd: create bench binary
This new binary will allow an easier benchmark on Powergate/FFS.

### LINKED ISSUE #376: ffs: trusted miners
Allow a config similar to `ExcludedMiners` but to indicate trusted miners.
These miners should always be used when making deals, falling back to the current logic if more results are needed in a query.

### PR DESCRIPTION
**Pull request by jsign**
- Closes #375 
- Closes #376 
- Update interopnet again.
- Multiple discovered bugs were fixed.
- Make commands renaming.

### CODE REVIEW

**Comment by asutula** [comment]
👀

File: exe/cli/cmd/ffs_watch.go [hunk 1]
```diff
@@ -131,18 +131,9 @@ func jobsComplete(state map[string]*client.JobEvent) bool {
 }
 
 func displayName(s ffs.JobStatus) string {
```
[the remark this answers is not in the document]
Comment (asutula) [reply exe/cli/cmd/ffs_watch.go:133]: 👍

File: ffs/scheduler/scheduler.go [hunk 2]
```diff
@@ -14,7 +14,7 @@ import (
 )
 
 const (
-	maxParallelExecutions = 10
+	maxParallelExecutions = 50
```
[the remark this answers is not in the document]
Comment (asutula) [reply ffs/scheduler/scheduler.go:17]: Oh, ok we do suport concurrent!

File: exe/bench/runner/runner.go [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+	}
+	ctx = context.WithValue(ctx, client.AuthKey, tok)
+	info, err := c.Ffs.Info(ctx)
+	if err != nil {
+		return fmt.Errorf("getting instance info: %s", err)
+	}
+	addr := info.GetInfo().GetBalances()[0].GetAddr().GetAddr()
+	time.Sleep(time.Second * 5)
+
+	chLimit := make(chan struct{}, ts.MaxParallel)
```
[the remark this answers is not in the document]
Comment (jsign) [reply exe/bench/runner/runner.go:79]: Yep, I think in a later comment you saw that we support concurrent execution within a single FFS instance. The thing is that at the moment that the concurrency limit is fixed, which would be easy to make it configurable anyway.

File: exe/bench/runner/runner.go [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+	}
+	var s client.JobEvent
+	for s = range chJob {
+		if s.Err != nil {
+			return fmt.Errorf("job watching: %s", s.Err)
+		}
+		log.Infof("[%d] Job changed to status %s", id, ffs.JobStatusStr[s.Job.Status])
+		if s.Job.Status == ffs.Failed {
+			return fmt.Errorf("job execution Failed")
+		}
```
[the remark this answers is not in the document]
Comment (jsign) [reply exe/bench/runner/runner.go:165]: Agree, will add.

**Comment by jsign** [comment]
Did some extra things since:
- Yesterday interopnet was reset again, so update it.
- Use the final go-ipfs:0.5.0 instead of latest rc.
- Covered the canceled status

**Comment by jsign** [comment]
Related #347, to keep a ref.

**Pull request merged**